### ISSUE TITLE
Inline ignores in files

### ISSUE THREAD

**Issue by jbergstroem** [rhysd/actionlint#237]
I'd love to see inline/annotation support for ignore rules. Many other linters (one example: shellcheck) does the same.

The ignore rule would then only apply to the following line instead of all files in the run. Similarly, applying it to the top of the file would only apply it to said file.

Example (from actionlint annotation examples):

```yaml
      - name: Run Actionlint
        # actionlint ignore=SC2016
        run: |
          actionlint -format '{{range $err := .}}::error file={{$err.Filepath}},line={{$err.Line}},col={{$err.Column}}::{{$err.Message}}{{end}}' .github/workflows/*.yml
```

**Comment by cooljeanius** [comment]
Slightly related, but I also tried putting the ignore comment directly into the script, and it turns out that that doesn't work either, i.e.:

```yaml
      - name: Build
        run: |
          # shellcheck disable=SC2001
          # (script goes here)
```
It would be nice if the existing format for shellcheck comments were supported in addition to any new formats.

**Comment by tueda** [comment]
Inline ignores would also help suppress false positives such as the one reported in https://github.com/rhysd/actionlint/issues/657 while https://github.com/rhysd/actionlint/pull/654 remains pending.
